collect_app/src/main/res/xml/preferences.xml

```diff
@@ -75,6 +75,12 @@
             android:key="font_size"
             android:title="@string/font_size" />
         <CheckBoxPreference
+            android:id="@+id/instance_sync"
+            android:defaultValue="true"
+            android:key="instance_sync"
```

Comment:
Yes. This is doable right? Since we need to keep this String in sync with the constant KEY_INSTANCE_SYNC, so I'm wondering if putting it in String resource would be better?